tests/test_logger.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+import os
+import o80_pam
+import pam_interface
+import tempfile
+import time
+
+class O80_PAM_LOGGER_TESTCASE(unittest.TestCase):
+
+    def test_exception_on_invalid_path(self):
+        try:
+            o80_pam.Logger("foo_id",os.path.join("not","existing"))
+            raised=False
+        except FileNotFoundError:
+            raised=True
+        # note: self.assertRaises did not work
+        self.assertTrue(raised)
```

Comment:
unittest has an `assertRaises` which can be used to simplify this:
```python
with self.assertRaises(FileNotFoundError):
    o80_pam.Logger("foo_id",os.path.join("not","existing"))
```

Reply:
That was my first version, for some unknown reason, it did not work (i.e. the test would not pass)